07LessonSeven/in-class-project/index.html

```diff
@@ -2,9 +2,41 @@
 <html>
 
 <head>
-    <title>Template</title>
+    <title>NASA page</title>
+    <link rel="stylesheet" href="css/my-style.css" />
 </head>
 
-<body></body>
+<body>
+<section id="sec1">
+  <h1>NASA</H1>
+</section>
+<section id="sec2">
+  <div id="div1">
+    <h2>Social Media</h2>
+    <p>Twitter: @nasa</p>
+    <p>Facebook: fb.com/nasa</p>
+    <p>Instagram: @nasa</p>
+    <p>YouTube: youtube.com/nasa</p>
+  </div>
+  <div id="div2">

```

Comment:
good initial structure, but remember to give yourself some vertical space for commenting. also, habitualize the beautifying of your code.